beluga_ros/tools/conversion_utils.py

```diff
@@ -0,0 +1,273 @@
+# Copyright 2024 Ekumen, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import h5py
+import yaml
+from typing import Dict, Optional
+from pathlib import Path
+import matplotlib.pyplot as plt
+from scipy.stats import multivariate_normal
+
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class DiscreteCell:
+    x: int
+    y: int
+
+
+@dataclass(frozen=True)
+class NormalDistribution:
+    """
+    Wrapper around scipy's multivariate normal distribution implementation.
+
+    Allows for equality checks and typing hints.
+    """
+
+    mean: np.ndarray
+    covariance: np.ndarray
+
+    def get_scipy_representation(self):
```

Comment:
@serraramiro1 nit^N!: what about `to_scipy` for consistency (with `to_hdf5`, with `to_numpy` in Pandas, etc.)?

Reply:
Done